testing/i3blocks/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=i3blocks
+pkgver=1.4
+pkgrel=0
+pkgdesc="A minimalist scheduler for your status line scripts"
+url="https://github.com/vivien/i3blocks"
+arch="all"
+license="GPL-3.0-or-later"
+options="!check"
```

Comment:
sadly missing explanation why...

Reply:
can you explain what you mean? that i used the version now instead of the master commit? or the missing check?

Reply:
I mean that `options=!check` needs comment why there's no tests or package can't be tested